### PR TITLE
Added Edit profile skeleton and sidebar

### PR DESCRIPTION
**Pull request by hiteshparate**
### What this PR does (required):
- Added Sidebar and profile Skeleton

### Screenshots / Videos (required):
- image

### Any information needed to test this feature (required):
- Add 'ProfileSetting' component in login page to test the feature

### CODE REVIEW

File: client/src/components/EditProfile/EditProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            <Grid item xs={6}>
+              <Select variant="outlined" label="Male" className={classes.formSelectField1}>
+                <MenuItem value="male">Male</MenuItem>
+                <MenuItem value="female">Female</MenuItem>
+                <MenuItem value="other">Other</MenuItem>
+              </Select>
+            </Grid>
+            <Grid item xs={2}></Grid>
+
+            {/* BirthDate  */}
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx:75]: This comment in not needed for production

File: client/src/components/EditProfile/EditProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            <Grid item xs={2}>
+              <Select variant="outlined" label="Male" className={classes.formSelectField2}>
+                <MenuItem value="male">Male</MenuItem>
+                <MenuItem value="female">Female</MenuItem>
+                <MenuItem value="other">Other</MenuItem>
+              </Select>
+            </Grid>
+            <Grid item xs={2}></Grid>
+
+            {/* email Address    */}
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx:105]: This too

File: client/src/components/EditProfile/EditProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+            <Grid item xs={2}></Grid>
+            <Grid item xs={2} className={classes.formLabel}>
+              EMAIL ADDRESS
+            </Grid>
+            <Grid item xs={6} className={classes.formInputField}>
+              <TextField size="small" variant="outlined" placeholder="johndoe@gmail.com" fullWidth></TextField>
+            </Grid>
+            <Grid xs={2}></Grid>
+
+            {/* Phone Number   */}
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx:115]: And here 👍🏻

File: client/src/components/EditProfile/EditProfile.tsx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+              PHONE NUMBER
+            </Grid>
+            <Grid item xs={6} className={classes.formInputField}>
+              <Button variant="outlined" className={classes.buttonPhone}>
+                Add a phone number
+              </Button>
+            </Grid>
+            <Grid xs={2}></Grid>
+
+            {/* Address */}
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx:127]: Here too

File: client/src/components/EditProfile/EditProfile.tsx [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+            <Grid item xs={2}></Grid>
+            <Grid item xs={2} className={classes.formLabel}>
+              WHERE YOU LIVE
+            </Grid>
+            <Grid item xs={6} className={classes.formInputField}>
+              <TextField size="small" variant="outlined" placeholder="johndoe@gmail.com" fullWidth></TextField>
+            </Grid>
+            <Grid xs={2}></Grid>
+
+            {/* Description  */}
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx:137]: And lastly, here. Thank you!

File: client/src/components/ProfileSettings/ProfileSettings.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+          }}
+          anchor="left"
+        >
+          <div className={classes.toolbar} />
+          <List>
+            {['Edit Profile', 'Profile Photo', 'Availability', 'Payment', 'Security', 'Settings'].map((text, index) => (
+              <ListItem button key={text}>
+                <ListItemText primary={text} className={classes.listItem} />
+              </ListItem>
+            ))}
```
Comment (moffatethan) [inline client/src/components/ProfileSettings/ProfileSettings.tsx:30]: You aren't using `index` so that isn't needed to be grabbed in the parameters.

**Review by moffatethan — APPROVED** [review]
Left you some comments to address but after this can be merged in. Good work!

**Comment by c-s-ale** [comment]
Hey Hitesh! Would it be possible to style the button for the phone number more like this: 
image ?

**Comment by hiteshparate** [comment]
> Hey Hitesh! Would it be possible to style the button for the phone number more like this:
> image ?

Thanks for reminding, I have added color in the Button.

File: client/src/components/EditProfile/EditProfile.tsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  phone?: string;
+  place?: string;
+  description?: string;
+}
+
+const EditProfile = (props: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <>
```
Comment (moffatethan) [inline client/src/components/EditProfile/EditProfile.tsx:23]: This fragment is not needed here as the Grid container wraps everything.

File: client/src/components/Sidebar/SideBar.tsx [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+import ListItemText from '@material-ui/core/ListItemText';
+import { Link } from 'react-router-dom';
+
+import useStyles from './useStyles';
+
+const Sidebar = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <>
```
Comment (moffatethan) [inline client/src/components/Sidebar/SideBar.tsx:13]: Same here, the fragment isn't necessary as the Drawer is wrapping everything.

**Review by moffatethan — APPROVED** [review]
LGTM! Left a few comments for some cleanup

**Pull request merged**